app/views/shared/_statements_table.html.haml

```diff
@@ -9,7 +9,7 @@
         %th= Statement.human_attribute_name(:sent_to)
         - unless @account
           %th= Account.model_name.human
-        - unless current_facility.try(:single_facility?)
+        - if current_facility&.cross_facility?
```

Comment:
I think this should now be `&.global?`, no? Or am I confused by `current_facility` (which is not an account)?

Reply:
`Facility#cross_facility?` and `Facility#single_facility?` are different from account. And they make sense (at least to me). `Facility#cross_facility?` returns true for the facility null object (when you're in the admin cross facility context). `single_facility?` returns true when you're in a single facility's context.

This was one of the first ones I caught going through the diff after a global find/replace and realized I replaced a bunch of `cross_facility?`s on Facilities that shouldn't had gotten applied.